sleap/gui/widgets/video.py

```diff
@@ -1039,7 +1041,7 @@ def mouseReleaseEvent(self, event):
         scenePos = self.mapToScene(event.pos())
 
         # check if mouse moved during click
-        has_moved = event.pos() != self._down_pos
+        has_moved = self._down_pos is not None and event.pos() != self._down_pos
```

Comment:
Hi @roomrys ,

Have we faced this issue before?
`small bug in GUI video that makes it crash when clicking.`

Reply:
The error I got was something like `self._down_pos is not defined`.
Before this PR, it was not defined in `__init__`, only when `mousePressEvent` is called for the first time. I somehow got the `mouseReleaseEvent` method called before `mousePressEvent` and triggered the bug.

Reply:
Hi @getzze,

sure, thank you for this!